src/List.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+describe('List 컴포넌트는', () => {
+  const handleClickDeleteTask = jest.fn();
+
+  context('task에 값이 없으면', () => {
+    const tasks = [];
+
+    it('default text가 표시된다.', () => {
+      const defaultText = '할 일이 없어요!'
+      const { container } = render((
+        <List
+          tasks={tasks}
+        />
```

Comment:
props가 하나 인 경우 한줄로 해주셔도 괜찮아요.
줄바꿈을 하는 기준이 있다는게 중요해요!

```suggestion
        <List tasks={tasks} />
```